lib/oat/adapters/siren.rb

```diff
@@ -91,7 +91,11 @@ class Field
           attr_reader :data
 
           def initialize(name)
-            @data = { :name => name }
+            @data = { :name => name, :class => []}
+          end
+
+          def class(value)
```

Comment:
You're overwriting Ruby's `#class` method, which is expected to return the object's class.